index.html

```diff
@@ -41,7 +41,18 @@ <h2 id="modified" style="text-align: center"></h2>
     <div class="info">
       Want to become a mentor?
       <a target="_blank" rel="noopener noreferrer" href="https://forms.gle/3VpkMX5jd9ZS55DS9">Sign up here</a>
+
+      <!-- Adding linkedin icon -->
+      <a alt="external link to linkedin" target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/company/bobatalks">
+        <img class="logo" src="assets/linkedin_logo.png" alt="LinkedIn logo" />
+      </a>
+
+      <!-- Adding discord icon -->
+      <a alt="external link to linkedin" target="_blank" rel="noopener noreferrer" href="https://discord.gg/bobatalks">
+        <img class="logo" src="assets/discord_icon.png" alt="Discord logo" />
+      </a>
```

Comment:
Hey @richardmbui thanks for trying to get these in, but these are part of another issue, so we should keep the scope of this PR focused on the page layout issue. Lets remove this for now.